### PR TITLE
Added a11y to the component

### PR DESCRIPTION
**Pull request by moaaz-bhnas**
I used this component in a form, and it wasn't focusable using the `tab` key, so I added another prop named `a11y`.  
If it's set to `true`, the component will be focusable, and users can use numbers and arrow keys to set its value.  
Lastly, thank you for adding "half" functionality. Helped me a lot.

### CODE REVIEW

**Comment by ertanhasani** [comment]
Thank you for this feature @moaaz-bhnas. There is some stuff not working. I ran `npm run dev-example` but there are some problems there:

-  The border when the component is focused does not look good (try to run it on a big screen and you see that its hovering `100%`, so CSS for that should be changed!) or maybe think of something else instead of a border!

- If count is more than 5, then its stopping at 5 when using numbers and arrow keys (see comment in file)

- Documentation `Readme.MD` needs to be updated

**Comment by moaaz-bhnas** [comment]
Thank u for the feedback. Will get back to u.

**Comment by moaaz-bhnas** [comment]
To to make the border, or whatever style will be applied, take the width of the component, and not 100%, I need to add another inner container to wrap the stars. Is that okay?

**Comment by ertanhasani** [comment]
> @ertanhasani
> To make the border, or whatever style will be applied, take the width of the component, and not 100%, I need to add another inner container to wrap the stars. Is that okay?

Feel free to do it.

File: src/react-stars.js [hunk 1]
```diff
@@ -190,6 +196,36 @@ class ReactStars extends Component {
...
+      }
+    } else { // string
+      if ((key === 'ArrowUp' || key === 'ArrowRight') && value < 5) {
+        event.preventDefault();
+        value += config.half ? 0.5 : 1;
+      } else if ((key === 'ArrowDown' || key === 'ArrowLeft') && value >.5) {
+        event.preventDefault();
+        value -= config.half ? 0.5 : 1;
+      }
+    }
```
[the remark this answers is not in the document]
Comment (moaaz-bhnas) [reply src/react-stars.js:217]: Thank you

**Comment by moaaz-bhnas** [comment]
I added another commit. Please review.  
Few notes:  
- I stopped the focus ring from taking up 100% of the screen, but i didn't change its style as I always prefer the default behavior of the browser because its predictable by users. However, if you prefer another style, let me know, and we can discuss it.
- Using keyboard numbers is limited to maimum of 9. I though of disabling numbers support, but actually most rating components are only 5, so I decided to leave that to you. If you think we should  remove keyboard numbers, I'll make another commit.

**Comment by moaaz-bhnas** [comment]
I'm sorry about my code formatter package changing your code style .. just noticed.

**Comment by ertanhasani** [comment]
> I added another commit. Please review.
> Few notes:
> 
> * I stopped the focus ring from taking up 100% of the screen, but i didn't change its style as I always prefer the default behavior of the browser because its predictable by users. However, if you prefer another style, let me know.
> * Using keyboard numbers is limited to maimum of 9. I though of disabling numbers support, but actually most rating components are only 5, so I decided to leave that to you. If you think we should  remove keyboard numbers, I'll make another commit.

I think thats ok!

**Pull request merged**